cypress-io/cypress
# feat: Support "Queued" latest run status

* Update GQL schema
* Add handling for `UNCLAIMED` status
* Only show loading indicator in Latest Runs section on initial fetch

### User facing changelog


### Additional details


### Steps to test
1. Open project and log in to cloud
2. Observe "latest run" data displays loading indicator on initial load of Specs Explorer
3. Start recording of project run in another terminal
4. Every 30 seconds app should poll for updated "latest run" data, observe that loading indicator is not shown during refetches
5. Mouse over the currently-running spec's "latest run" column, observe tooltip with "Queued" status

### How has the user experience changed?
https://user-images.githubusercontent.com/11482842/175427837-387cd142-0926-4ce4-8647-fe8125e70d2f.mov

### PR Tasks

- [na] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
